xcode/Subconscious/Shared/Models/Slashlink.swift

```diff
@@ -16,7 +16,7 @@ public struct Slashlink:
     Codable,
     LosslessStringConvertible
 {
-    private static let slashlinkRegex = /(\@(?<petname>(?:[\w\d\-]+)(?:\.[\w\d\-]+)*))?(\/(?<slug>(?:[\w\d\-]+)(?:\/[\w\d\-]+)*))?/
+    private static let slashlinkRegex = /(?:(?<did>(?:did:key:[a-zA-Z0-9]+))|(?<petname>@[\w\-]+(?:\.[\w\-]+)*))?(?<slug>\/[\w\-]+(?:\/[\w\-]+)*)?/
```

Comment:
This regex matches the following forms:

- `[did][slug]`
- `[petname][slug]`
- `[slug]`
- (nothing)

For simplicity, I am being strict right now and only matching `did:key`.